repose-aggregator/components/filters/translation-filter/src/test/java/org/openrepose/filters/translation/TranslationFilterMethodsTest.java

```diff
@@ -0,0 +1,123 @@
+import org.openrepose.commons.utils.servlet.http.ResponseMode;
+import org.openrepose.core.services.config.ConfigurationService;
+import org.openrepose.filters.translation.config.*;
+import org.springframework.mock.web.MockFilterConfig;
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.mock.web.MockHttpServletResponse;
+import org.xml.sax.SAXException;
+
+import java.io.IOException;
+
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.springframework.http.HttpHeaders.ACCEPT;
+import static org.springframework.http.MediaType.APPLICATION_XML_VALUE;
+import static org.xmlunit.matchers.CompareMatcher.isSimilarTo;
+
+public class TranslationFilterMethodsTest {
+    private static byte[] contentRemoveMe;
+
+    private TranslationFilter filter;
+    private MockHttpServletRequest mockRequest;
+    private MockHttpServletResponse mockResponse;
+    private HttpServletRequestWrapper httpServletRequestWrapper;
+    private HttpServletResponseWrapper httpServletResponseWrapper;
+
+    @BeforeClass
+    public static void setupSpec() throws Exception {
+        contentRemoveMe = IOUtils.toByteArray(TranslationFilterMethodsTest.class.getResourceAsStream("/remove-me-element.xml"));
+    }
+
+    @Before
+    public void setup() throws Exception {
+        ConfigurationService configurationService = mock(ConfigurationService.class);
+        String configurationRoot = "";
+        filter = new TranslationFilter(configurationService, configurationRoot);
+
+        MockFilterConfig mockFilterConfig = new MockFilterConfig("TranslationFilter");
+        filter.init(mockFilterConfig);
+
+        TranslationConfig config = new TranslationConfig();
+
+        StyleSheets sheets = new StyleSheets();
+        StyleSheet sheet = new StyleSheet();
+        sheet.setId("sheet1");
+        sheet.setHref("classpath:///identity.xsl");
+        sheets.getStyle().add(sheet);
+
+        sheet = new StyleSheet();
+        sheet.setId("sheet2");
+        sheet.setHref("classpath:///add-element.xsl");
+        sheets.getStyle().add(sheet);
+
+        sheet = new StyleSheet();
+        sheet.setId("sheet2");
+        sheet.setHref("classpath:///remove-element.xsl");
+        sheets.getStyle().add(sheet);
+
+        RequestTranslations requestTranslations = new RequestTranslations();
+        RequestTranslation trans1 = new RequestTranslation();
+        trans1.setAccept(APPLICATION_XML_VALUE);
+        trans1.setContentType(APPLICATION_XML_VALUE);
+        trans1.setTranslatedContentType(APPLICATION_XML_VALUE);
+        trans1.setStyleSheets(sheets);
+        trans1.getHttpMethods().add(HttpMethod.POST);
+        requestTranslations.getRequestTranslation().add(trans1);
+
+        ResponseTranslations responseTranslations = new ResponseTranslations();
+        responseTranslations.getResponseTranslation().add(new ResponseTranslation());
```

Comment:
This was a copy/pasta from the other tests and wasn't completely removed. Updated.